misc/ERD.json

```diff
@@ -0,0 +1,64 @@
+// users
+{
+  "user_id": 1,
+  "username": "agao",
+  "password": "12345",
+  "full_name": "Andy Gao",
+  "email": "agao@andrew.cmu.edu",
+  "address": "5023 Forbes Ave",
+  "phone_number": 9011234567,
+  "birthday": 12041997,
+  "favorite_donut": "pineapple",
+  "credit_card": 1234567898765432
+}
+// customers
+{
+  "customer_id": 1,
+  "user_id": 1,
+  "address": "5023 Forbes Ave",
+  "birthday": 12041997,
+  "fav_donut": "pineapple",
+  "credit_card": 1234567898765432
+}
+// employees
+{
+  "employee_id": 1,
+  "user_id": 1,
+  "position": "manager"
+}
+// order
+{
+  "order_id": 1,
+  "user_id": 1,
+  "employee_id": 1,
+  "donut": [
+    "pineapple",
+    "watermelon"
+  ],
+  "time": "2012-11-04 14:55:45",
+  "drone_id": 1
```

Comment:
Should we also store the order "status"? For example, the order could be "received" (by the donut shop), "out for delivery", "delivery completed", etc.